presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -11,8 +11,13 @@ The Iceberg connector allows querying data stored in Iceberg tables.
 
     It is recommended to use Iceberg 0.9.0 or later.
 
-Configuration
--------------
+Metastores
+-----------
+Iceberg tables stores most of the metadata in the metadata files along with the data on the filesystem.
+But to store small amount of metadata it still requires Metastore to find the current location of the current
+metadata pointer for a table. This central place is called ``Iceberg Catalog``.
+The Iceberg connector supports different types of Iceberg Catalogs : ``Hive Metastore``,
+``GLUE``, ``NESSIE``, ``HADOOP``
 
 To configure the Iceberg connector, create a catalog properties file
```

Comment:
I think it is better to highlight the `iceberg.catalog.type` property here. Would you please add a sentence for it?